docs/guide/acquiring_data/jsoc.rst

```diff
@@ -0,0 +1,295 @@
+the VSO Client.
+
+There are two ways of downloading JSOC data. One way is using Sunpy's unified search interface,
+known as FIDO. FIDO supplies a single, easy and consistent way to to obtain most forms of physics
+data. An alternative way to fetch data from JSOC is by using the underlying JSOC Client. This option
+can be preferred when the complex searches are to be made, or when you need to separate the staging
+and downloading steps, which is not supported by FIDO.
+
+The JSOC stages data before you can download it,
+so a JSOC query is a three stage process, first you query the JSOC for records,
+a table of these records is returned. Then you can request these records to be
+staged for download and then you can download them. Fido combines the stages into 2, ``search()``
+and ``fetch()``.
+
+Setup
+-----
+
+SunPy's FIDO module is in ``sunpy.net``.  It can be imported as follows:
+
+    >>> from sunpy.net import Fido, attrs as a
+
+The JSOC client handles the particulars of how the data from
+the data provider is downloaded to your computer.
+
+Querying the JSOC
+-----------------
+
+To search for data in JSOC, your query needs at minimum a series name and a primekey.
+Different primekeys are supported by different series, and you can find out the primekeys
+supported in any series by:
+
+	>>> import drms
+	>>> c = drms.Client()
+	>>> print(c.pkeys('hmi.m_720s'))
+
+The most common primekey, that is supported by every series is Time, that is denoted by
+T_REC or T_OBS. Hence, Time can always be passed as an attribute while building a query.
+Wavelength is another pre-defined attributes which is a prime-key.
+Other primekeys which needs to be passed, should be manually passed in PrimeKey(). This
+will be explained later in detail.
+
+Constructing a Basic Query
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Let's start with a very simple query.  We could ask for all `hmi.v_45s` series data
+between January 1st from 00:00 to 01:00, 2014.
+
+    >>> res = Fido.search(a.jsoc.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00'), a.jsoc.Series('hmi.v_45s'))
+
+This returns an `~sunpy.net.fido_factory.UnifiedResponse` object containing
+information on the available online files which fit the criteria specified by
+the attrs objects in the above call. It does not download the files.
+
+To see a summary of results of our query, simple type the name of the
+variable set to the Fido search, in this case, res::
+
+    >>> res
+
+Now, let's break down the arguments of ``Fido.search`` to understand
+better what we've done.  The first argument:
+
+    ``a.jsoc.Time('2014-01-01T00:00:00', '2014-01-01T01:00:00')``
+
+sets the start and end times for the query (any date/time
+format understood by SunPy's :ref:`parse_time function <parse-time>`
+can be used to specify dates and time). The Time attribute takes UTC time,
+as default. If you need to pass a Time in some other time scale, such as TAI,
+pass an Astropy Time object, like:
+
+	>>> from astropy.time import Time as T
```

Comment:
Using Time would collide with Time attribute of Sunpy. I faced a bit of problem there.